raspberryPI3/ros2_ws/src/geicar_start/launch/geicar.launch.py

```diff
@@ -52,6 +52,12 @@ def generate_launch_description():
         executable="system_check_node",
         emulate_tty=True
     )
+    
+    rosbridge_server_node = Node(
+        package="rosbridge_server",
+        executable="rosbridge_websocket",
```

Comment:
Dumb question but why we don't see the package itself on your PR ?

Reply:
When using ROS, you source 2 files: ROS2 Humble and your workspace. In your environment variables, ROS_PACKAGES_PATH (or something similar) has at least two paths: ROS2 Humble and your workspace. It means that you can either use packages from your workspace or from ROS2 Humble. So, if you install a package on the system using `apt install ros-humble-the-package-from-ros-depo`, it will be installed in the ROS2 Humble path and not in your workspace. 
You can then directly use this package in your workspace or in the terminal. In our workspace, we tell the system that we are using this package in `package.xml` (that is what I have done in commit with id: 7979d72). \

So when we setup our environment we can say to ros that we depend on this package and it will automatically download it and will be able to use it.

We usually do that when we want to use a package from the Internet without modifying it.